src/index.js

```diff
@@ -0,0 +1,264 @@
+
+let locationError = false;
+let tempType = "fahrenheit";
+let count = 72;
+const defaultCity = "Seattle";
+
+cityNameInput.defaultValue = defaultCity;
+headerCityName.textContent = defaultCity;
+
+const updateLandscape = (tempValue) => {
+    let landscape_text = "";
+    if (tempType === "fahrenheit") {
+        if (tempValue >= 86) {
+            landscape_text = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+        } else if (tempValue >= 71 && tempValue <= 85) {
+            landscape_text = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+        } else if (tempValue >= 56 && tempValue <= 70) {
+            landscape_text = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+        } else if (tempValue >= 40 && tempValue <= 55) {
+            landscape_text = "💨🍃🪵_🍁🍂_🍁_🍃🪵🍂💨";
+        } else if (tempValue <= 39) {
+            landscape_text = "🌲🌲⛄️🌲⛄️☃️🌲☃️🌲🌲⛄️☃️🌲";
+        }
+    } else {
+        if (tempValue >= 33) {
+            landscape_text = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+        } else if (tempValue >= 22 && tempValue <= 32) {
+            landscape_text = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+        } else if (tempValue >= 13 && tempValue <= 21) {
+            landscape_text = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+        } else if (tempValue >= 6 && tempValue <= 12) {
+            landscape_text = "💨🍃🪵_🍁🍂_🍁_🍃🪵🍂💨";
+        } else if (tempValue <= 5) {
+            landscape_text = "🌲🌲⛄️🌲⛄️☃️🌲☃️🌲🌲⛄️☃️🌲";
+        }
+    }
+
+    landscape.textContent = landscape_text;
+};
+
+const updateTextColor = (tempValue) => {
+    if (tempType === "fahrenheit") {
+        if (tempValue >= 80) {
+            document.getElementById("tempValue").style.color = "red";
+        } else if (tempValue >= 70 && tempValue <= 79) {
+            document.getElementById("tempValue").style.color = "orange";
+        } else if (tempValue >= 60 && tempValue <= 69) {
+            document.getElementById("tempValue").style.color = "yellow";
+        } else if (tempValue >= 50 && tempValue <= 59) {
+            document.getElementById("tempValue").style.color = "green";
+        } else if (tempValue >= 30 && tempValue <= 49) {
+            document.getElementById("tempValue").style.color = "teal";
+        } else if (tempValue <= 29) {
+            document.getElementById("tempValue").style.color = "blue";
+        }
+    } else {
+        if (tempValue >= 30) {
+            document.getElementById("tempValue").style.color = "red";
+        } else if (tempValue >= 23 && tempValue <= 29) {
+            document.getElementById("tempValue").style.color = "orange";
+        } else if (tempValue >= 16 && tempValue <= 22) {
+            document.getElementById("tempValue").style.color = "yellow";
+        } else if (tempValue >= 10 && tempValue <= 15) {
+            document.getElementById("tempValue").style.color = "green";
+        } else if (tempValue >= 1 && tempValue <= 9) {
+            document.getElementById("tempValue").style.color = "teal";
+        } else if (tempValue <= 0) {
+            document.getElementById("tempValue").style.color = "blue";
+        }
+    }
+};
```

Comment:
Another way to format these two methods (maybe combine them into one) would be to have an object that has temp values as keys, and then you could have values be a list. The list could hold background color and background image, like:
```javascript
const tempDisplayObject = {
    80: ['red',  "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"], 
    70: ['orange', "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"]
} 
```
When you have an object, you don't need to do if/else if/else if/ else which can introduce bugs and also adds complexity to the method. If you have an object, than you can iterate over the object and check the keys with `tempValue` and then get the correct values to set the color and landscape emojis.